vir/test.cpp

```diff
@@ -31,6 +32,20 @@ static_assert(vir::iota_v<V<int>>[0] == 0);
 static_assert(all_of(vir::iota_v<V<short>> == V<short>([](short i) { return i; })));
 #endif // VIR_HAVE_SIMD_IOTA
 
+#if VIR_HAVE_SIMD_CVT
+namespace
+{
+  using vir::cvt;
+  // simd:
+  static_assert(all_of(cvt(V<int>(2)) * V<float>(1) == V<float>(2)));
+  static_assert(all_of(cvt(10 * V<float>(cvt(V<int>(2)))) == V<int>(20)));
+  // simd_mask:
+  static_assert(all_of(cvt(V<int>(2) == 2) == (V<float>(1) == 1.f)));
```

Comment:
In the example / docs you're using `stdx::rebind_simd_t`.
If I udnerstand [the docs](https://en.cppreference.com/w/cpp/experimental/simd/rebind_simd) correctly, this keeps the number of elements? (I assume, that for example `stdx::rebind_simd_t<double, stdx::native_simd<int>>` will not work?)

Here you're using `stdx::native_simd<T>` all around, right? If I understood that correctly, then `stdx::native_simd<int>` and `stdx::native_simd<double>` can (and usually) have different number of elements?

Please explain this (probably in a comment in the code and/or docs).